packages/salesforcedx-vscode-core/package.json

```diff
@@ -803,6 +803,11 @@
           "type": "boolean",
           "default": false,
           "description": "%experimental_deploy_retrieve_description%"
+        },
+        "salesforcedx-vscode-core.experimental.cliCommand": {
```

Comment:
We can have this under experimental, change `cliCommand` to `apexlibrary` or something apex specific and default it to true. 

Or we can change the name to be nothing under experimental and keep it as something along the lines of default to cli command use.